### PR TITLE
scripts for database creation(drop) with test data

### PR DESCRIPTION
**Pull request by NewIvanIvanov**

### CODE REVIEW

File: createDB.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+#Credentals for PostgresSQL Server. You mast have superuser with 'login' nickname and 'pass' password.
+dbCredentals = 'postgresql+psycopg2://login:pass@localhost/test'
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = dbCredentals
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
+
+
+class Attachment(db.Model):
```
Comment (Michaluch) [inline createDB.py]: please move each class in separate file

**Review by Michaluch — CHANGES_REQUESTED** [review]

File: models.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    image_url = db.Column(db.Text)
+    delete_date = db.Column(db.Date)
+
+    issue = db.relationship(u'Issue')
+
+
+class Category(db.Model):
+    """This class is used for Category table in database."""
+
+    __tablename__ = 'Category'
```
Comment (andrewRv) [inline models.py]: Table name in lowercase

File: models.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    """This class is used for User table in database."""
+
+    __tablename__ = 'Users'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.Text)
+    alias = db.Column(db.Text)
+    email = db.Column(db.Text)
+    password = db.Column(db.Text)
+    role_id = db.Column(db.ForeignKey(u'Roles.id'), index=True)
```
Comment (andrewRv) [inline models.py]: Add cascade deleting

File: createDB.py (hunk 1 above)
Comment (NewIvanIvanov) [reply createDB.py]: move classes into seperate file

File: models.py (hunk 2 above)
Comment (NewIvanIvanov) [reply models.py]: rename tables in lowercase

File: models.py (hunk 3 above)
Comment (NewIvanIvanov) [reply models.py]: not sure about that. need to discuss

**Pull request closed without merging**